test-workspaces/sfdx-multi-workspace/sfdx-workspace-1/force-app/main/default/lwc/todo/store.js

```diff
@@ -0,0 +1,9 @@
+const STORAGE_KEY = 'todos';
```

Comment:
Do we run automated tests on each one of these components? If not we should probably delete some of them